pkg/webhooks/webhooks.go

```diff
@@ -74,14 +82,44 @@ func (wh WebhookHandler) updateOperatorCr(ctx context.Context, hc *v1beta1.Hyper
 
 	switch obj := exists.(type) {
 	case *kubevirtv1.KubeVirt:
-		existingKv := obj
+		existing := obj
```

Comment:
Actually, we don't need this variable. But I wouln't hold the PR for that. (same note for all the cases)